docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -137,447 +108,216 @@ If you are using an older version, please upgrade.
             
             Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
             permitted by applicable law.
-            root@ev3dev:~# 
+            robot@ev3dev:~$ 
+
+*   {: tab="Ubuntu"}<span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+    This instructions were written using Ubuntu 15.10 using the default desktop
```

Comment:
Although it may be a more accurate expression of your inner feelings, "this instructions were written" is not a well-formed phrase :wink: